apertium_apy/apy.py

```diff
@@ -19,6 +19,12 @@
 import sys
 from datetime import timedelta
 from logging import handlers as logging_handlers  # type: ignore
+try:
+    import pkg_resources
+    found = True
```

Comment:
This variable name is way too general. Instead, just do `pkg_resources = None` in the other branch.

Reply:
Okay I have removed found variable

Reply:
Why is this back?

Reply:
Actually the problem is we cannot assign a module name to NoneType object in Python 3.5, 3.6. Travis build passes only in python 3.4. So we need another variable. Is it fine?

Reply:
It seems to work here: https://github.com/apertium/apertium-apy/blob/master/apertium_apy/handlers/translate_webpage.py#L15 ?

Reply:
I tried it but the build fails every time.

Reply:
It works perfect on my local machine but doesn't pass travis ci